lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+<%%= f.error_notification %>
+
+<div class="form-inputs">
+  <%- attributes.each do |attribute| -%>
+  <%%= f.<%= attribute.reference? ? :association : :input %> :<%= attribute.name %> %>
+  <%- end -%>
+</div>
+
+<div class="form-actions">
+  <%%= f.button :submit %>
```

Comment:
do you need a translation for the submit button?

Reply:
Yes I did all the translations but it's in the other PR, I was trying to get this the most possible "just style".. ;) But thanks for all the comment! :)